src/pages/search/SearchPage.tsx

```diff
@@ -17,19 +17,25 @@ import { RegistrationSearchBar } from './registration_search/RegistrationSearchB
 import { SearchTypeField, SearchTypeValue } from './SearchTypeField';
 import { SelectedPersonFacetsList } from './selected_facets/SelectedPersonFacetsList';
 import { SelectedProjectFacetsList } from './selected_facets/SelectedProjectFacetsList';
+import { SelectedResultFacetsList } from './selected_facets/SelectedResultFacetsList';
 
 const StyledSearchBarContainer = styled(Box)(({ theme }) => ({
   display: 'grid',
   gridTemplateColumns: 'auto 1fr',
   [theme.breakpoints.down('sm')]: {
     gridTemplateColumns: '1fr',
   },
-
   gap: '0.75rem 0.5rem',
+}));
+
+const StyledContainer = styled(Box)(({ theme }) => ({
+  display: 'flex',
+  flexDirection: 'column',
+  gap: '0.75rem',
 
-  margin: '0 0 1rem 0',
+  marginBottom: '0.75rem',
   [theme.breakpoints.down('md')]: {
-    margin: '0 0.5rem 1rem 0.5rem',
+    margin: '0 0.5rem 0.75rem 0.5rem',
   },
 }));
```

Comment:
Denne kunne kanskje hatt et mer beskrivende navn, og blitt flytta til `Wrappers.tsx`? Ser vi har en del forskjellig styled components liggende rundt omkring, men det hadde hjulpet med å holde komponentene mindre? Det samme gjelder forsåvidt den over også

Reply:
Ja, tenkte litt på navnet der men fant ikke noe bedre egentlig 🤔 Ble litt krøkk av at det er to styled containers her, hvor en bare passer på margin, mens den andre er container for søketype og search bar.. Noen forslag?

Men tenker i i grunn det er like greit at den ligger lokalt her, da disse er spesifikt for disse. Ellers blir det fort at samme Styled component importeres inn til flere ulike stedet, og når vi skal endre en liten ting på en av dem får vi ikke med oss hvordan det påvirker ting alle andre plasser. Blir mer isolert og enklere å ha oversikt når den bare ligger i komponenten den brukes da

Reply:
Nei, har egentlig ikke noe bedre forslag til navn her🤔 men det går veil greim om den kun skal brukes lokalt